ast/ast.go

```diff
@@ -1497,7 +1520,7 @@ type CreateSequence struct {
 
 	Name        *Ident
 	IfNotExists bool
-	Options     *SequenceOptions
+	Options     *Options
```

Comment:
```suggestion
	Options     *Options // optional
```

Also, we should update the document comment for this struct.

Reply:
Repeat:

> we should update the document comment for this struct.

Reply:
I have noticed that the current implementation still treat `OPTIONS` clause as required.
https://github.com/apstndb/memefish/blob/1e59f4158f2626d063128898c42fbd804f6c60b8/parser.go#L2183

It is because there is required option `sequence_kind`, so `OPTIONS` is not optional.
> The `sequence_kind` option accepts a `STRING` to indicate the type of sequence to use. At this time, `bit_reversed_positive` is the only valid type and it's a required option.

```
spanner> CREATE SEQUENCE MySequence;
ERROR: rpc error: code = InvalidArgument desc = MySequence does not have a valid sequence kind. Please specify the sequence kind explicitly or set the database option `default_sequence_kind`.
```

We can parse `OPTIONS` as optional to ready future changes, should it be?

Reply:
Ok, I understand that `Options` should be required here, so let's revert it.

But, repeat,

> we should update the document comment for this struct.

Currently,

```go
//	CREATE SEQUENCE {{if .IfNotExists}}IF NOT EXISTS{{end}} {{.Name | sql}} }} OPTIONS ({{.Options | sqlJoin ","}})
```

but the following is corret I think:

```go
//	CREATE SEQUENCE {{if .IfNotExists}}IF NOT EXISTS{{end}} {{.Name | sql}} {{.Options | sql}}
```

Reply:
Thank you!
I believe it is correct changes https://github.com/cloudspannerecosystem/memefish/pull/142/commits/405a7c5bdcc5bd676116697d6522642dfb5326a4.